grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/Reasoner.java

```diff
@@ -162,10 +160,10 @@ public static void precomputeInferences(GraknGraph graph){
         }
         Set<VarName> selectVars = inputQuery.admin().getSelectedNames();
         Iterator<Conjunction<VarAdmin>> conjIt = inputQuery.admin().getPattern().getDisjunctiveNormalForm().getPatterns().iterator();
-        ReasonerQueryImpl conjunctiveQuery = new ReasonerMatchQuery(graph.graql().match(conjIt.next()).select(selectVars), graph);
+        ReasonerQueryImpl conjunctiveQuery = new ReasonerQueryImpl(graph.graql().match(conjIt.next()).select(selectVars), graph);
         Stream<Map<VarName, Concept>> answerStream = conjunctiveQuery.resolve(materialise);
         while(conjIt.hasNext()) {
-            conjunctiveQuery = new ReasonerMatchQuery(graph.graql().match(conjIt.next()).select(selectVars), graph);
+            conjunctiveQuery = new ReasonerQueryImpl(graph.graql().match(conjIt.next()).select(selectVars), graph);
```

Comment:
No, at the moment you have two entry points to reasoning (aiming to reduce it to one), one through graql and one through Reasoner class, both require match queries. Reasoner queries stopped being match queries a couple of commits ago. The renaming took place  in the previous commit. This commit consolidates the 4 reasoner queries into 2, that's the key change.